src/js/themes/base.js

```diff
@@ -1029,12 +1030,20 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           color: 'status-disabled',
           cursor: 'default',
         },
+        pinned: {
+          color: 'background-contrast',
+          icon: {
```

Comment:
Can we not rely on our t-shirt sizing for these?

Reply:
@ericsoderberghp, I tried using the t-shirt sizing, but the medium is too big and small is too small in my opinion. I found that 18px was around the same size as the arrows for the non-pinned items and the 15px padding made it the same size (44px) box to keep everything consistent at all screen sizes. Below are some screenshots for reference to show the differences. Let me what you think and how you'd like me to proceed.

---

#### size: 18px
![image](https://user-images.githubusercontent.com/7104326/185145801-040e8002-4566-4602-bebc-1a0b6867a128.png)

---

#### size: 'medium'
![image](https://user-images.githubusercontent.com/7104326/185145938-5b39a10c-3f09-4522-99e6-0937103e65e9.png)

---

#### size: 'small'
![image](https://user-images.githubusercontent.com/7104326/185146449-fb922ae5-cf50-4a64-b428-50d6540fb52a.png)

Reply:
To align the icon sizes better, I think we should add a "FormPin" icon, to go with "FormUp" and "FormDown" being used. This would involve creating a new svg and adding it to `grommet-icons`.

Reply:
@ericsoderberghp, okay no problem. I've opened a [PR](https://github.com/grommet/grommet-icons/pull/283) over on the grommet-icons repo to add in this icon. Once that's merged I'll reference that in here.
Would I update the grommet-icons version in the package.json or is there a separate process to follow for that?

Reply:
Hey @jeffcap1 thanks for creating the grommet-icons PR. We need to release a new version of grommet-icons so that we are able to use the new icon. I created a ticket for the team to release a new version of grommet icons that can be tracked here: https://github.com/grommet/hpe-design-system/issues/2842

Reply:
@ericsoderberghp, this has now been updated to use the new FormPin icon from the latest grommet-icons release. Please let me know if you see anything else that needs to be updated.